Broken cross-compilation without dejavu font

Cross-compilation is broken since commit 1d66106e5ae45407b01459cb112ee09752166dba. Indeed, when cross-compiling, the dejavu font won't be installed in /usr/share/fonts. The path (e.g. `$(TARGET_DIR)/usr/share/fonts/dejavu/DejaVuSans.ttf`) could be provided through `--with-test-font-path`  which will fix the build failure but this will result in broken test programs. As dejavu font is only needed for tests, it could be useful to revert this change or add a --disable-test option

The build failure are raised by the buildroot autobuilders since we bumped to version 1.27.5: http://autobuild.buildroot.org/?reason=cups-filters-1.27.5 so it includes #240.

This issue is raised when the build machine does not have any `/usr/share/fonts/ ` directory:
```
find: `/usr/share/fonts': No such file or directory
configure: error: DejaVuSans.ttf font file is missing. Please install a package providing it.
```
or when `DejaVuSans.ttf` can't be found in `/usr/share/fonts/`:
`configure: error: DejaVuSans.ttf font file is missing. Please install a package providing it.`

On buildroot side, we're not really interested in building or running tests as we're targetting embedded devices so an option to disable this check would be nice, I tried to set `--with-test-font-path` to `/dev/null` but this does not work as `/dev/null` is not a standard file. An other option would to hardcode TESTFONT to "/usr/share/fonts/dejavu/DejaVuSans.ttf" if cross_compiling is set. Would it be acceptable? If this is the case, I can send a PR.

--- a/configure.ac
+++ b/configure.ac
@@ -796,10 +796,10 @@ AC_ARG_WITH([test-font-path],
 	[with_test_font_path=`find /usr/share/fonts -name DejaVuSans.ttf || echo /usr/share/fonts/dejavu/DejaVuSans.ttf`]
 )
 
-AS_IF([test -f "$with_test_font_path"],
-	[AC_DEFINE_UNQUOTED([TESTFONT], ["$with_test_font_path"], [Path to font used in tests])],
+AS_IF([test "x$cross_compiling" != "xyes" && ! test -f "$with_test_font_path"],
 	[AC_MSG_ERROR(DejaVuSans.ttf font file is missing. Please install a package providing it.)]
 )
+AC_DEFINE_UNQUOTED([TESTFONT], ["$with_test_font_path"], [Path to font used in tests])
 
 # ================
 # Check for cflags

